src/test/resources/insert/annotation/NEW_Annotation.java

```diff
@@ -0,0 +1,10 @@
+interface INothing {
+    public void doNothing();
+}
```

Comment:
What's the purpose of the interface for this test case?

Reply:
I wanted to use the annotation `@Override` and since we compile our test resources, I had to define an interface or some class.

As of now, classpath is not provided to the script so we have limited options for using annotations.

Reply:
You can just override any method in `Object`. Like, `Object.toString()`

Reply:
Yeah, better idea. I shall do that.